### PR TITLE
feat: enable back to original size after exit fullscreen.

### PR DESCRIPTION
**Pull request by re2zero**
After exit fullscreen, it still show as fullscreen with title, this make user feel unwell. Record the normal size and then recover will be beter.

Log: support recover size.

### CODE REVIEW

**Comment by barry-ran** [comment]
I didn't reproduce this problem. Can you record a video to describe it

**Comment by re2zero** [comment]
Some screenshots:
image

image

File: QtScrcpy/ui/videoform.cpp [hunk 1]
```diff
@@ -467,6 +467,8 @@ void VideoForm::switchFullScreen()
         }
 
         showNormal();
+        // back to normal size.
```
Comment (barry-ran) [inline QtScrcpy/ui/videoform.cpp:470]: 我明白了，修改下注释吧，某些平台（ubuntu）需要此操作，因为我之前一直测试windows没问题

Comment (re2zero) [reply QtScrcpy/ui/videoform.cpp:470]: 现在github不能直接在这里修改代码吗？想直接就在这里修改提交，我记得是可以呀，是不是没有点“Request changes”？

**Review by barry-ran — APPROVED** [review]

**Pull request merged**